src/Og.php

```diff
@@ -169,6 +171,31 @@ public static function getEntityGroups($entity_type, $entity_id, $states = [OG_S
   }
 
   /**
+   * Return TRUE if entity belongs to a group.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $group
+   *   The group entity to check.
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity to get groups for.
+   * @param array $states
+   *   (optional) Array with the state to return. If empty groups of all state
```

Comment:
`(optional) Array with the membership states to check the membership. Defaults to active memberships.`